csharp/device/Microsoft.Azure.Devices.Client/DeviceClientHelper.cs

```diff
@@ -39,15 +47,28 @@ protected DeviceClientHelper()
         /// <summary>
         /// Explicitly open the DeviceClient instance.
         /// </summary>
+#if NETMF
+        public void Open()
+        {
+            this.EnsureOpened(true);
+        }
+#else
         public Task OpenAsync()
         {
             return this.EnsureOpenedAsync(true);
         }
+#endif
 
         /// <summary>
         /// Close the DeviceClient instance
         /// </summary>
         /// <returns></returns>
+#if NETMF
+        public void Close()
+        {
+
```

Comment:
Yes. Because there is nothing to close... Left it there in order to keep 100% compatibility with the API.